contracts/PreseasonTwo/BuilderNFTPreSeason02Implementation.sol

```diff
@@ -490,4 +519,9 @@ contract BuilderNFTPreSeason02Implementation is
     function acceptUpgrade() public view returns (address) {
         return address(this);
     }
+
+    function _validateMint(address account, uint256 tokenId) internal view {
```

Comment:
I would also replace the first 2 lines of mint() method with this, so validation behaviour is the same between mint and mintTo